### PR TITLE
added: play next to video and  music context menus

### PR DESCRIPTION
**Pull request by notspiff**
## Description
Add 'Play next' to context menu

## Motivation and Context
Tadly asked for it.

## How Has This Been Tested?
Barely

### CODE REVIEW

File: xbmc/music/windows/GUIWindowMusicBase.cpp [hunk 1]
```diff
@@ -519,6 +522,7 @@ void CGUIWindowMusicBase::GetContextButtons(int itemNumber, CContextButtons &but
       if (item->CanQueue() && !item->IsAddonsPath() && !item->IsScript())
```
Comment (da-anda) [inline xbmc/music/windows/GUIWindowMusicBase.cpp]: have not tested this PR yet, but I wonder if `!item->IsAddonsPath()` is still valid here? We have music add-ons that could also benefit from this, like the googleMusic add-ons. @tadly mind testing this with your gmusic add-on while testing the PR?

Comment (tadly) [reply xbmc/music/windows/GUIWindowMusicBase.cpp]: Sure, will do

**Comment by tadly** [comment]
Tested and confirmed working:
  * [x] Local video
  * [x] Local music
  * [x] Video addon (youtube) 
  * [x] Music addon (gmusic)

File: addons/resource.language.en_gb/resources/strings.po [hunk 2]
```diff
@@ -2258,7 +2258,10 @@ msgctxt "#489"
 msgid "Play next song automatically"
 msgstr ""
 
-#empty string with id 490
+#: xbmc/music/windows/GUIWindowMusicBase.cpp
```
[the remark this answers is not in the document]
Comment (MartijnKaijser) [reply addons/resource.language.en_gb/resources/strings.po]: Please a higher string ID (above 10000)

File: xbmc/music/windows/GUIWindowMusicBase.cpp (hunk 1 above)
Comment (DaveTBlake) [reply xbmc/music/windows/GUIWindowMusicBase.cpp]: Yes I think it remains a valid test. It is excluding addons themselves, not items that happen to have been added to the item list by an addon.

**Pull request closed without merging**